tools/hicup/hicup_truncater.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="hicup_truncater" name="Truncater" version="0.1.0">
+    <macros>
+        <import>hicup_macros.xml</import>
+    </macros>
+    <expand macro="requirements_hicup" />
+    <expand macro="stdio" />
+    <command><![CDATA[
+        export FONTCONFIG_FILE=\$CONDA_PREFIX/etc/fonts/fonts.conf &&
+        export FONTCONFIG_PATH=\$CONDA_PREFIX/etc/fonts/ &&
```

Comment:
This should really be part of the conda package and not the Galaxy recipe.
https://github.com/bioconda/bioconda-recipes/blob/2f69f1286bed04cf12638ce84189556ce28ebff9/recipes/rnasnp/scripts/activate.sh